### DISCUSSION TITLE
[Question] # PyArmor Plugin

### DISCUSSION THREAD

**Discussion by VeNoMouS** [dashingsoft/pyarmor#1303] [Q&A] [answered]
In 7.x etc... using `# PyArmor Plugin:` we could hide / disable / active code when code was run via pyarmor .. 

is there such a feature in 8.x?

**Comment by jondy** [comment, marked as answer]
Yes, please check https://pyarmor.readthedocs.io/en/stable/tutorial/advanced.html

**Comment by VeNoMouS** [comment]
ah cheers, [exact url here](https://pyarmor.readthedocs.io/en/stable/tutorial/advanced.html#patching-source-by-inline-marker)

thanks
